openapi/index.yaml

```
@@ -188,6 +218,99 @@ components:
+    NotificationPhysicalAddress:
+      description: "Indirizzo fisico scoperto durante fase di consegna"
+      type: "object"
+      required:
+        - "address"
+        - "municipality"
+        - "zip"
+      properties:
+        at:
+          type: "string"
+          description: "Campo \"presso\" dell'indirizzo"
+        address:
+          type: "string"
+          description: "Indirizzo del domicilio fisico"
+        addressDetails:
+          type: "string"
+          description: "Seconda riga dell'indirizzo fisico"
+        zip:
+          type: "string"
+          description: "Codice di avviamento postale"
+        municipality:
+          type: "string"
+          description: "Comune"
+        municipalityDetails:
+          type: "string"
+          description: "Frazione o località"
+        province:
+          type: "string"
+          description: "Provincia"
+        foreignState:
+          type: "string"
+          description: "Denominazione paese estero"
+    NotificationPaymentInfo:
+      title: Informazioni per effettuare il pagamento
+      description: >-
+        Informazioni utili per effettuare il pagamento di una notifica, sono associate al 
+        destinatario perché le spese di notifica possono differire a seconda del canale di 
+        notifica utilizzato. <br/>
+          - _notificationFeePolicy_: indica se il pagamento delle spese di notifica è forfettario o 
+             calcolato puntualmente. <br/>
+          - _noticeCode_: "codice avviso pagoPA" di pagamento del sistema pagoPA, usato per pagamento online.<br/>
+          - _creditorTaxId_: codice fiscale dell'ente a cui fa riferimento il "codice avviso pagoPA". <br/>
+          - _pagoPaForm_: riferimento al PDF contenete il bollettino pagoPA<br/>
+          - _f24flatRate_: riferimento al PDF contenete l'F24 con spese di notifica forfettarie<br/>
+          - _f24standard_: riferimento al PDF contenete l'F24 con spese di notifica standard<br/>
+      type: object
+      required:
+        - noticeCode
+        - creditorTaxId
+      properties:
+        noticeCode:
+          $ref: '#/components/schemas/noticeCode'
+        creditorTaxId:
+          $ref: '#/components/schemas/paTaxId'
+        noticeCodeAlternative:
+          $ref: '#/components/schemas/noticeCode'
+        pagoPaForm:
+          $ref: "#/components/schemas/NotificationPaymentAttachment"
+        f24flatRate:
+          $ref: "#/components/schemas/NotificationPaymentAttachment"
+        f24standard:
+          $ref: "#/components/schemas/NotificationPaymentAttachment"
+    noticeCode:
+      description: "Payment notice number  numero avviso"
+      example: "302000100000019421"
+      type: "string"
+      maxLength: 18
+      minLength: 18
+      pattern: "^\\d+$"
+    paTaxId:
+      description: "Payment PA fiscal code"
+      example: "77777777777"
+      type: "string"
+      maxLength: 11
+      minLength: 11
+      pattern: "^\\d+$"
```

They must be defined in the following ways in order to be generated :/

```suggestion
    noticeCode:
      $ref: 'https://raw.githubusercontent.com/pagopa/pn-delivery/0599f1287a08c315bc20410fc57a666e0fa6f79d/docs/openapi/schemas-pn-notification-v1.yaml#/components/schemas/noticeCode'
    paTaxId:
      $ref: 'https://raw.githubusercontent.com/pagopa/pn-delivery/0599f1287a08c315bc20410fc57a666e0fa6f79d/docs/openapi/schemas-pn-notification-v1.yaml#/components/schemas/paTaxId'
```